app/helpers/foreman_host_reports/reports_dashboard_helper.rb

```diff
@@ -26,10 +26,12 @@ def host_reports_get_overview(options = {})
         nochange_hosts: _('No Change'),
         disabled_hosts: _('Disabled alerts'),
       }
-
+      data = state_labels.map { |key, label| [label, send(key, format), host_reports_report_color[key]] }
+      total = data.reduce(0) { |sum, arr| sum + arr[1] }
```

Comment:
Would it be better to do something like:
```rb
total = []
data = state_labels.map |key, label| do
  count = send(key, format)
  total.push(count)
  [label, count, host_reports_report_color[key]]
end
```
And later you could also use it in the title's value, e.g: `count[1]`

Reply:
Do you mean instead of `(send(:failure_hosts, format)`? 
I am not sure it is more readable

Reply:
Just to save another call, but not mandatory..